ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/RpcApi.java

```diff
@@ -45,6 +44,6 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return MoreObjects.toStringHelper(this).add("cliValue", cliValue).toString();
+    return cliValue;
```

Comment:
I think this is probably a place where a comment is worth while explaining that the CLI help output uses `toString()` as it's very unexpected otherwise (the test will pick it up but nice to get feedback sooner than that).